jwst/regtest/test_miri_mrs.py

```diff
@@ -32,6 +32,25 @@ def run_spec2(jail, rtdata_module):
     return rt.run_step_from_dict(rtdata_module, **step_params)
 
 
+@pytest.fixture(scope='module')
+def run_spec3(jail, rtdata_module):
```

Comment:
This pipeline doesn't piggyback off any of the data in the spec2 pipeline?  I.e. is there any dataset that runs all the way through levels 2 and 3?

Reply:
Will look into that.